package/kernel/bcm63xx-cferam/Makefile

```diff
@@ -0,0 +1,42 @@
+include $(TOPDIR)/rules.mk
+include $(INCLUDE_DIR)/kernel.mk
+
+PKG_NAME:=bcm63xx-cferam
+PKG_VERSION:=2020-05-10
+PKG_RELEASE:=1
+
+PKG_SOURCE_URL:=https://github.com/Noltari/bcm63xx-cferam.git
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_VERSION:=bc48a2657ec5a4a3fec88bb6d3bedca5de6c062f
+PKG_MIRROR_HASH:=d584cbfb15acacb8e64cbc93e55663e43aa769955580f364d57a1135402143b7
+
+PKG_BUILD_DIR:=$(KERNEL_BUILD_DIR)/$(PKG_NAME)
```

Comment:
If you don't use PKG_VERSION, I think this can be dropped.

Reply:
Ok